internal/services/netapp/netapp_account_resource.go

```diff
@@ -194,6 +202,10 @@ func resourceNetAppAccountCreate(d *pluginsdk.ResourceData, meta interface{}) er
 		Tags:       tags.Expand(d.Get("tags").(map[string]interface{})),
 	}
 
+	if nfsv4IDDomain := d.Get("nfsv4_id_domain").(string); nfsv4IDDomain != "" {
```

Comment:
Because this has a default, we know it'll never be empty so we can just set it in the struct above